keyboards/delikeeb/vaneelaex/v/config.h

```diff
@@ -0,0 +1,162 @@
+/*
```

Comment:
Everything in this `v` folder should be moved one level up.

Reply:
Ahh.. this was a mistake while I moving things around. This will be deleted.

Reply:
v folder is deleted in the latest commit.